src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,7 @@
 	<uses-permission android:name="com.android.launcher.permission.INSTALL_SHORTCUT"/>
 	<uses-permission android:name="com.android.launcher.permission.UNINSTALL_SHORTCUT"/>
 	<!-- READ_EXTERNAL_STORAGE required if users want to include photos from their phone -->
-	<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
You don't need both?

Reply:
No, `WRITE` implies `READ`.  Comment above looks wrong now, though :~)